src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -105,6 +105,7 @@ public boolean hasCustomer(Customer customer) {
     @Override
     public void deleteCustomer(Customer target) {
         addressBook.deleteCustomer(target);
+        setPanelToCustomerList();
```

Comment:
Would you be double calling the method (because you call it in Delete___Command.java as well). Would suggest that you call it from outside instead of here.

Possible use case is, in the future if we allow deletion of customer with all of the customer's order, the panel should only change to the customer panel and not the order panel